cabby/data/extract_test.py

```diff
@@ -14,15 +14,13 @@
 
 '''Tests for wikidata.py'''
```

Comment:
Long before this PR. But just noticed it chould be chainged.

Reply:
This test failed when I ran it.

Reply:
Fixed the comment, good catch!

The test runs fine for me. Can you try doing the following?

```
$ bazel clean
$ ./build_all.sh
$ ./test_all.sh
```